### ISSUE TITLE
GLTracker::initializeGL: Fix clang-analyzer-optin.cplusplus.VirtualCall warning

### ISSUE THREAD

**Issue by jcfr** [BrownBiomechanics/Autoscoper#91]
The call was initially introduced in 6b7ad8e. At the time of the original commit, the base class was `QGLWidget` and calling the virtual function likely didn't have the intended effect.

### Warning

For context, the warning was reported using the `Analyze > Clang-Tidy and Clazy` feature available within Qt Creator. See https://doc.qt.io/qtcreator/creator-clang-tools.html

```
/home/jcfr/Projects/Autoscoper/autoscoper/src/ui/GLTracker.cpp:66:3: Call to virtual method 'GLTracker::initializeGL' during construction bypasses virtual dispatch [clang-analyzer-optin.cplusplus.VirtualCall]
```

### Proposed solution

* Remove commented code from the constructor originally introduced in 9cc8c2928
* Remove the call to `initializeGL()`

### References

* https://doc.qt.io/qt-5/qopenglwidget.html#initializeGL
* https://doc.qt.io/qt-5/qglwidget.html#initializeGL
* https://releases.llvm.org/14.0.0/tools/clang/tools/extra/docs/clang-tidy/checks/clang-analyzer-optin.cplusplus.VirtualCall.html
* https://clang.llvm.org/docs/analyzer/checkers.html#optin-cplusplus-virtualcall-c
